scripts/run_e2e_tests.py

```diff
@@ -536,7 +536,6 @@ def run_tests(args):
 
     with contextlib2.ExitStack() as stack:
         stack.enter_context(common.managed_elasticsearch_dev_server())
-        stack.enter_context(common.managed_firebase_auth_emulator())
```

Comment:
I haven't removed this from the `start.py` script, should I remove that too? (Just to replicate the same behavior on the local server and the CI tests?) If yes, then I'm assuming that I've to also remove the `managed_firebase_auth_emulator` unused function from the codebase.

Reply:
I would probably say so to remove it from there, whether to leave the unused method in the codebase I think I would let @brianrodri decide.

Reply:
Yes removing it everywhere makes the most sense to me. I'll re-add it when we're in a safer place.